lib/plugins/create/templates/aws-scala-sbt/src/main/scala/hello/ApiGatewayResponse.scala

```diff
@@ -3,4 +3,4 @@ package hello
 import scala.beans.BeanProperty
 
 case class ApiGatewayResponse(@BeanProperty statusCode: Integer, @BeanProperty body: String,
-    @BeanProperty headers: java.util.Map[String, Object], @BeanProperty base64Encoded: Boolean = false)
+    @BeanProperty headers: java.util.Map[String, String], @BeanProperty base64Encoded: Boolean = false)
```

Comment:
Changed return type to `String`, but no problem.  
API Gateway expects `Map[String, String]` for `header`.

https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/APIGatewayProxyResponseEvent.java#L15